conda/meta.yaml

```diff
@@ -35,6 +35,7 @@ requirements:
     - transformers
     - pandas
     - pip
+    - python-facebook-api
```

Comment:
Not sure if this released on conda channel. If not then we need to add this in `-pip` block same as `presidio-analyzer`